src/api/app/controllers/source_controller.rb

```diff
@@ -294,15 +294,7 @@ def lastevents
 
   # POST /source?cmd=createmaintenanceincident
   def global_command_createmaintenanceincident
-    # set defaults
-    at = nil
-    unless params[:attribute]
-      params[:attribute] = 'OBS:MaintenanceProject'
-      at = AttribType.find_by_name!(params[:attribute])
-    end
-
-    # find maintenance project via attribute
-    prj = Project.get_maintenance_project!(at)
+    prj = Project.get_maintenance_project!
```

Comment:
I don't have enough information to be 100% sure, but maybe the right thing is to fix the conditional to be able to use the `params[:attribute]` instead of removing the code?

Reply:
Hm... Not sure if we ever had a use case for that...

Reply:
Alright, delete all the code! :boom: